sdk/recoveryservices-datareplication/Azure.ResourceManager.RecoveryServicesDataReplication/src/autorest.md

```diff
@@ -0,0 +1,49 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: RecoveryServicesDataReplication
+namespace: Azure.ResourceManager.RecoveryServicesDataReplication
+require: D:\forked-repos\openapihub\azure-rest-api-specs\specification\recoveryservicesdatareplication\resource-manager\readme.md
```

Comment:
point to the public repo instead of local file path

Reply:
Started using https://github.com/Azure/azure-rest-api-specs/blob/64cddc58b25ae7d6e50cc0dfb5bc19af4ea23f65/specification/recoveryservicesdatareplication/resource-manager/Microsoft.DataReplication/preview/2021-02-16-preview/recoveryservicesdatareplication.json